frontend/src/app/projects/models/create-model/create-model.component.html

```diff
@@ -5,54 +5,55 @@
 
 <div class="wrapper">
   <mat-horizontal-stepper
+    class="w-[680px]"
     (selectionChange)="onStepChange($event)"
     linear
     #stepper
   >
-    <mat-step completed="false" label="Create model">
-      <app-create-model-base
-        [asStepper]="asStepper"
-        (create)="afterModelCreated()"
-      ></app-create-model-base>
-    </mat-step>
+    <div class="flex justify-center">
```

Comment:
The component is not centered when triggered via the project details: 
![image](https://github.com/DSD-DBS/capella-collab-manager/assets/23395732/58ac7af9-c3d4-4ef7-8f10-913335d8d4a9)

![image](https://github.com/DSD-DBS/capella-collab-manager/assets/23395732/27a62f10-d287-41e1-ae2b-efed97d6432c)

Reply:
This change got lost with the last push, the model creation dialog has no shadow / rounded anymore: 

![image](https://github.com/DSD-DBS/capella-collab-manager/assets/23395732/9b804187-ca3e-4090-a35c-71d9153ad433)


Also, the width of model description and model tool is not full anymore.

Reply:
Waah, replace-all error (the styling was actually still there but set to be sm instead of md), now fixed. Full-width applied to inputs again.